.github/workflows/pull_request.yml

```diff
@@ -95,6 +95,7 @@ jobs:
         # Ensure we can upgrade the CRD with the current changes
         make install
         kubectl wait --for condition="established" crd --all
+        sleep 60
```

Comment:
Can wee make a comment here that this is a workaround and we should investigate how to solve this without a sleep?